api/environments/identities/models.py

```diff
@@ -1,23 +1,17 @@
 import typing
 
-import boto3
-from django.conf import settings
 from django.db import models
 from django.db.models import Prefetch, Q
 from django.utils.encoding import python_2_unicode_compatible
-from flag_engine.identities.builders import build_identity_dict
 
+from environments.dynamodb import DynamoIdentityWrapper
 from environments.identities.traits.models import Trait
 from environments.models import Environment
 from features.models import FeatureState
 from features.multivariate.models import MultivariateFeatureStateValue
 
-# Intialize the dynamo client globally
-dynamo_identity_table = None
-if settings.IDENTITIES_TABLE_NAME_DYNAMO:
-    dynamo_identity_table = boto3.resource("dynamodb").Table(
-        settings.IDENTITIES_TABLE_NAME_DYNAMO
-    )
+# Initialize the dynamo wrapper client globally
+dynamo_identity_wrapper = DynamoIdentityWrapper()
```

Comment:
Why initialise this here only to import it in `serializers`? We should either initialise it in the `serializers` module or add the logic back to the model class here (just not using the manager). We could use class methods or similar?